===================================================================
RCS file: /cvsroot/plptools/plptools/lib/Enum.h,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- plptools/plptools/lib/Enum.h	2003/02/05 07:59:08	1.9
+++ plptools/plptools/lib/Enum.h	2004/03/13 11:15:36	1.10
@@ -27,6 +27,7 @@
 #include <config.h>
 #endif
 
+#include <assert.h>
 #include <map>
 #include <string>
 
@@ -276,13 +277,13 @@
  * // declaration of enumeration; somewhere
  * class rfsv {
  * [...]
- * 	enum PSI_ERROR_CODES { E_PSI_GEN_NONE, E_PSI_GEN_FAIL, E_PSI_GEN_ARG };
+ *	enum PSI_ERROR_CODES { E_PSI_GEN_NONE, E_PSI_GEN_FAIL, E_PSI_GEN_ARG };
  * [...]
  * };
  *
  * // definition of the Enum<E> with the appropriate string representations
  * ENUM_DEFINITION(rfsv::PSI_ERROR_CODES,
- *	 	   rfsv::E_PSI_GEN_NONE) {
+ *		   rfsv::E_PSI_GEN_NONE) {
  *	stringRep.add(rfsv::E_PSI_GEN_NONE,	"no error");
  *	stringRep.add(rfsv::E_PSI_GEN_FAIL,	"general");
  *	stringRep.add(rfsv::E_PSI_GEN_ARG,	"bad argument");
